### ISSUE TITLE
Fix Maven warnings regarding missing plugin versions

### ISSUE THREAD

**Issue by ansgarkonermann** [alexruiz/fest-assert-2.x#66]
Maven 3.x complains:

``` shell
/usr/java/jdk1.7.0_02/bin/java -Dclassworlds.conf=/home/ansgar/opt/apache-maven-3.0.4/bin/m2.conf -Dmaven.home=/home/ansgar/opt/apache-maven-3.0.4 -Didea.launcher.port=7532 -Didea.launcher.bin.path=/home/ansgar/opt/idea-IU-117.117/bin -Dfile.encoding=UTF-8 -classpath /home/ansgar/opt/apache-maven-3.0.4/boot/plexus-classworlds-2.4.jar:/home/ansgar/opt/idea-IU-117.117/lib/idea_rt.jar com.intellij.rt.execution.application.AppMain org.codehaus.classworlds.Launcher --no-plugin-registry --fail-fast --no-plugin-updates --strict-checksums --update-snapshots -f /home/ansgar/git-repositories/easytesting.org/github/fest-assert-2.x/pom.xml clean verify
[WARNING] Command line option -npu is deprecated and will be removed in future Maven versions.
[WARNING] Command line option -npr is deprecated and will be removed in future Maven versions.
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.easytesting:fest-assert-core:jar:2.0M7-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-resources-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 189, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-compiler-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 196, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-surefire-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 212, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-site-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 205, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-source-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 222, column 15
[WARNING] 'reporting.plugins.plugin.version' for org.apache.maven.plugins:maven-site-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 314, column 15
[WARNING] 'reporting.plugins.plugin.version' for org.codehaus.mojo:javancss-maven-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 318, column 15
[WARNING] 'reporting.plugins.plugin.version' for org.apache.maven.plugins:maven-javadoc-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 322, column 15
[WARNING] 'reporting.plugins.plugin.version' for org.codehaus.mojo:jdepend-maven-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 350, column 15
[WARNING] 'reporting.plugins.plugin.version' for org.codehaus.mojo:findbugs-maven-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 354, column 15
[WARNING] 'reporting.plugins.plugin.version' for org.apache.maven.plugins:maven-surefire-report-plugin is missing. @ org.easytesting:fest:1.0.11, /home/ansgar/.m2/repository/org/easytesting/fest/1.0.11/fest-1.0.11.pom, line 358, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
´´´

Should be fixed. 

What needs to be done:
- add plugin versions to parent POM
- test the new parent POM works great for FEST-Assert (install snapshot + build FEST-Assert)
- release parent POM
- update parent pom of FEST-Assert to new parent version

While we're at it: 
- we could update the plugin versions to the latest released versions
- plus review the plugins for redundant phase bindings (like source/javadoc plugin).
```

**Comment by joel-costigliola** [comment]
Ansgar,

I already fixed those warnings in fest parent pom 1.0.13, I see in the logs you are using 1.0.11 version, are you sure you are up to date ?

**Comment by joel-costigliola** [comment]
Ansgar, is it ok to close this issue ?
